app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -210,6 +210,19 @@ def track_events(otp_delivery_preference:)
         telephony_response: @telephony_result.to_h,
         success: @telephony_result.success?,
       )
+
+      # Should reauthentication fire a different attempt event?
+      if UserSessionContext.authentication_context?(context) || UserSessionContext.reauthentication_context?(context)
```

Comment:
Do we want to track a OTP sent sign-in event when the user is asked to reauthenticate? Wondering if the OR condition should be used here, or should this be a separate event all together?

Reply:
If we wanted to, we could add a `reauthentication: ` boolean maybe? I wouldn't break out a separate event

Reply:
Agreed. Will implement this.